doc/sphinx-guides/source/user/account.rst

```diff
@@ -155,6 +155,8 @@ The My Data section of your account page displays a listing of all the Dataverse
 
 You can use the Add Data button to create a new Dataverse collection or dataset. By default, the new Dataverse collection or dataset will be created in the root Dataverse collection, but from the create form you can use the Host Dataverse collection dropdown menu to choose a different Dataverse collection, for which you have the proper access privileges. However, you will not be able to change this selection after you create your Dataverse collection or dataset.
 
+.. _account-notifications:
+
```

Comment:
I'd just like to point out that below very few notifications are listed here in the User Guide:

"You will receive a notification when:
- You've created your account.
- You've created a Dataverse collection or added a dataset.
- Another Dataverse installation user has requested access to restricted files in a dataset that you published. (If you submitted your dataset for review and it was published by a curator, the curators of the Dataverse collection that contains your dataset will get a notification about requests to access your restricted files.)
- A file in one of your datasets has finished the ingest process."

The new docs in the Admin Guide list many more.

There's also no mention of email notifications. The focus seems to be in-app notifications.

Not sure what we should do about this.